File: programming/labaratory-work-1/solutions/solution-8/main.c
#include <stdio.h>

int main() {
  int n = 0;

  scanf_s("%d", &n);

  int eng = 0;
  int ger = 0;
  int fra = 0;

  for (int i = n; i > 0; i--) {
    int ans = 0;

    scanf_s("%d", &ans);

    switch (ans) {
    case 1:
      eng++;
      break;

    case 2:
      ger++;
      break;

    case 3:
      fra++;
      break;
    }
  }

  printf_s("%d %d %d", eng, ger, fra);

  return 0;
}

File: programming/labaratory-work-1/solutions/solution-8/go/main.go
package main

import "fmt"

func main() {
	var n int

	fmt.Scanf("%d", &n)

	var eng int
	var ger int
	var fra int

	for i := n; i > 0; i-- {
		var ans int

		fmt.Scanf("%d", &ans)

		switch ans {
		case 1:
			eng++
		case 2:
			ger++
		case 3:
			fra++
		}
	}

	fmt.Printf("%d %d %d", eng, ger, fra)
}